tools/ratbagctl.body.py.in

```diff
@@ -1386,10 +1386,9 @@ class RatbagParserSwitch(RatbagParser):
             if switch is not None
             else []
         )
-        if N_access is not None:
-            self.N_access = RatbagParserNAccess(**N_access)
-        else:
-            self.N_access = None
+        self.N_access = (
+            RatbagParserNAccess(**N_access) if N_access is not None else None
+        )
```

Comment:
> I think this would be better without the `()` so it doesn't look like a tuple at first glance

black does this to reduce line length, and I notice it quite often actually.

> One way to get around this btw is with something like this

Honestly, I don't agree with this. I think constuctor function should take non-optional arguments, and not have an optional return type unless the constructor actually does some fallible logic. (also I don't know how to type hint kwargs :smiley: I think mypy or pyright complain for them). Either way, this is only done once, so I don't see a need for a helper for this.

Reply:
> black does this to reduce line length, and I notice it quite often actually.

You can increase the line length. I use 127, which is (was?) the size of the github viewer.

> Honestly, I don't agree with this. I think constuctor function should take non-optional arguments, and not have an optional return type unless the constructor actually does some fallible logic.

I agree, no need for that kind of complexity here. We'd be better to just stick this an if clause here.